libs/components/datetime/src/lib/modules/date-range-picker/date-range-picker.component.html

```diff
@@ -91,7 +93,7 @@
     </sky-input-box>
   </div>
   <div
-    class="sky-date-range-picker-form-group sky-date-range-datepicker-wrapper"
+    class="sky-date-range-picker-form-group sky-date-range-datepicker-wrapper sky-date-range-picker-end-date"
```

Comment:
Instead of adding classes just for the harness, could you check for "hidden" by checking the required state instead? The datepicker inputs are marked as "required" when they're visible.
```
public async isStartDateVisible(): Promise<boolean> {
  const input = await (
    await this.#getStartDateInputBoxHarness()
  ).querySelector('input');

  const required = await input?.getProperty('required');

  return required === true;
}
```

Reply:
this class i added is also to get the individual datepickers to set the value, not just to check for hidden